tutorials/tour/_posts/2017-02-13-operators.md

```diff
@@ -10,8 +10,40 @@ num: 30
 next-page: automatic-closures
 previous-page: local-type-inference
 ---
+In Scala, operators are methods. For example, `+` can be called with dot-notation:
```

Comment:
You never actually say that methods can be used as operators!  This is **the most important** piece of information in this section!  Please add it again!